sdk/aqueduct/api_client.py

```diff
@@ -274,7 +275,16 @@ def get_workflow(self, flow_id: str) -> GetWorkflowResponse:
         url = self._construct_full_url(self.GET_WORKFLOW_ROUTE_TEMPLATE % flow_id, self.use_https)
         resp = requests.get(url, headers=headers)
         utils.raise_errors(resp)
-        return GetWorkflowResponse(**resp.json())
+
+        workflow_response = GetWorkflowResponse(**resp.json())
+        for work_flow_dag in list(workflow_response.workflow_dags.values()):
+            for operator in list(work_flow_dag.operators.values()):
+                operator_url = self._construct_full_url(
+                    self.EXPORT_FUNCTION_ROUTE % str(operator.id), self.use_https
+                )
+                operator_resp = requests.get(operator_url, headers=headers)
+                operator.change_file(operator_resp.content)
```

Comment:
instead of `change_file()` on the operator, I'd like to keep our current abstraction where the dag class is the only thing that can modify underlying operator and artifact objects. Having fewer places where things can modify the dag will make things easier to maintain + us to have fewer bugs.

Would you be able to use `dag.update_operator_spec()` instead? You can probably write a helper to construct the new operator spec from the old one + serialized contents.

Reply:
@kenxu95 Since `get_workflow` returns a class `WorkflowDagResponse`, the serialize function is inserted into `WorkflowDagResponse` rather than the `dag` class. Therefore I believe the update_operator_spec() should be under  `WorkflowDagResponse`?